specs/_features/maxeb_increase/capella.py

```diff
@@ -273,7 +275,7 @@ class Configuration(NamedTuple):
     SHARD_COMMITTEE_PERIOD=uint64(256),
     ETH1_FOLLOW_DISTANCE=uint64(2048),
     EJECTION_BALANCE=Gwei(16000000000),
-    MIN_PER_EPOCH_CHURN_LIMIT=uint64(4),
+    MIN_PER_EPOCH_CHURN_LIMIT=Gwei(1280000000000),
```

Comment:
Consider not changing this constant, instead just multiplying it by `MIN_ACTIVATION_BALANCE` whereever it is used.